src/api/Migrations/20181204172354_FasterLocationQuery.cs

```diff
@@ -0,0 +1,92 @@
+﻿using Microsoft.EntityFrameworkCore.Migrations;
+
+namespace SearchAndCompare.Migrations
+{
+    public partial class FasterLocationQuery : Migration
+    {
+        protected override void Up(MigrationBuilder migrationBuilder)
+        {
+            migrationBuilder.Sql(@"DROP FUNCTION course_distance(double precision,
+                                                                 double precision,
+                                                                 double precision);");
+
+            migrationBuilder.Sql(@"CREATE OR REPLACE FUNCTION course_distance( lat DOUBLE PRECISION, 
+                                                            lon DOUBLE PRECISION, 
+                                                            rad DOUBLE PRECISION) 
+                                            RETURNS TABLE (""Id"" integer,
+                                                           ""Distance"" double precision,
+                                                           ""DistanceAddress"" text) AS $$
+                    WITH distances as (
+                        SELECT ""Id"", ""Address"",
+                            111045.0* DEGREES(ACOS(COS(RADIANS(lat))
```

Comment:
please document your source for magic numbers and functions. Use sql comments so they make it all the way through to the db in case anyone is looking in there